lib/ui/BUILD.gn

```diff
@@ -12,6 +12,9 @@ source_set("ui") {
     "dart_runtime_hooks.h",
     "dart_ui.cc",
     "dart_ui.h",
+    "logger/logger.cc",
+    "logger/logger.h",
+    "logger/system_logger.h",
```

Comment:
Seems like this code should be in a lower layer somewhere.  Doesn't seem specific to the UI library.